lb/cross_reg_int_app_lb_mig_backend/main.tf

```diff
@@ -0,0 +1,384 @@
+  purpose       = "GLOBAL_MANAGED_PROXY"
+  region        = "us-west1"
+  role          = "ACTIVE"
+  lifecycle {
+    ignore_changes = [ipv6_access_type]
+  }
+}
+# [END cloudloadbalancing_cross_region_proxy_subnet_a_tag]
+
+# forwarding-rule in subnet_a
+# [START cloudloadbalancing_cross_region_fwd_rule_a_tag]
+resource "google_compute_global_forwarding_rule" "fwd_rule_a" {
+  provider              = google-beta
+  depends_on            = [google_compute_subnetwork.proxy_subnet_a]
+  ip_address            = "10.1.2.99"
+  ip_protocol           = "TCP"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  name                  = "gil7forwarding-rule-a"
+  network               = google_compute_network.default.id
+  port_range            = "80"
+  target                = google_compute_target_http_proxy.default.id
+  subnetwork            = google_compute_subnetwork.subnet_a.id
+}
+# [END cloudloadbalancing_cross_region_fwd_rule_a_tag]
+
+# forwarding-rule in subnet_b
+# [START cloudloadbalancing_cross_region_fwd_rule_b_tag]
+resource "google_compute_global_forwarding_rule" "fwd_rule_b" {
+  provider              = google-beta
+  depends_on            = [google_compute_subnetwork.proxy_subnet_b]
+  ip_address            = "10.1.3.99"
+  ip_protocol           = "TCP"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  name                  = "gil7forwarding-rule-b"
+  network               = google_compute_network.default.id
+  port_range            = "80"
+  target                = google_compute_target_http_proxy.default.id
+  subnetwork            = google_compute_subnetwork.subnet_b.id
+}
+# [END cloudloadbalancing_cross_region_fwd_rule_b_tag]
+
+# Target HTTP proxy
+# [START cloudloadbalancing_cross_region_tgt_http_proxy_tag]
+resource "google_compute_target_http_proxy" "default" {
+  name     = "gil7target-http-proxy"
+  provider = google-beta
+  url_map  = google_compute_url_map.default.id
+}
+# [END cloudloadbalancing_cross_region_tgt_http_proxy_tag]
+
+# URL map
+# [START cloudloadbalancing_cross_region_url_map_tag]
+resource "google_compute_url_map" "default" {
+  name            = "gl7-gilb-url-map"
+  provider        = google-beta
+  default_service = google_compute_backend_service.default.id
+}
+# [END cloudloadbalancing_cross_region_url_map_tag]
+
+# backend service
+# [START cloudloadbalancing_cross_region_bck_service_tag]
+resource "google_compute_backend_service" "default" {
+  name                  = "gl7-gilb-backend-service"
+  provider              = google-beta
+  protocol              = "HTTP"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  timeout_sec           = 10
+  health_checks         = [google_compute_health_check.default.id]
+  backend {
+    group           = 
+google_compute_region_instance_group_manager.mig_a.instance_group
+    balancing_mode  = "UTILIZATION"
+    capacity_scaler = 1.0
+  }
+  backend {
+    group           = 
+google_compute_region_instance_group_manager.mig_b.instance_group
```

Comment:
This line may need lint formatting here.

Reply:
we cannot fit the variable in the 80 character limit. kindly ignore this error